Changes for harmattan

CODeRUS opened the pull request:
Enabled booster (MDeclarativeCache)
Starting QNetworkSession if not opened before loading url
QML UI based on harmattan Qt Components

CODeRUS committed:
Changes for harmattan
Enabled booster (MDeclarativeCache)
Starting QNetworkSession if not opened before loading url
QML UI based on harmattan Qt Components

romaxa commented:
is this UI implementation compatible with non harmattan QML? can we make it working on desktop too?

CODeRUS commented:
no. based on harmattan Qt Components only.

CODeRUS committed:
Merge git://github.com/tmeshkova/mozilla-central into embedlite

CODeRUS committed:
LongTap event function getting link href from point

CODeRUS committed:
Merge git://github.com/tmeshkova/mozilla-central into embedlite

CODeRUS committed:
Context menu options and browser rect

CODeRUS committed:
Fixies
for https://github.com/tmeshkova/mozilla-central/pull/6#commitcomment-2521309
for https://github.com/tmeshkova/mozilla-central/pull/6#commitcomment-2521353
